app/components/sections/users/user-table/component.js

```diff
@@ -0,0 +1,48 @@
+import Ember from "ember";
+import computed from "ember-computed-decorators";
+import UserValidations from "last-strawberry/validators/user";
+import Roles from "last-strawberry/constants/roles";
+
+export default Ember.Component.extend({
+  session: Ember.inject.service(),
+
+  classNames: "col stretch",
+
+  roles: _.values(Roles),
```

Comment:
Is there a reason to wrap this in lodash? We should be able to expose it by doing roles: Roles. Let me know what happened when you tried that.